futures/README.md

```diff
@@ -43,7 +43,7 @@ fn main() {
       // connect() returns a future of an AMQP Client
       // that resolves once the handshake is done
       lapin::client::Client::connect(stream, &ConnectionOptions::default())
-    }).and_then(|client| {
+    }).and_then(|(client, )| {
```

Comment:
and shouldn't that be `(client, _)` to ignore it instead? Or is that a syntax I didn't know about